Mathlib/LinearAlgebra/RootSystem/Finite/Nondegenerate.lean

```diff
@@ -348,6 +348,54 @@ lemma rootSpan_eq_top_iff :
   · rw [← P.finrank_corootSpan_eq', h, finrank_top, P.toPerfPair.finrank_eq,
       Subspace.dual_finrank_eq]
 
+section IsRootSystem
+
+variable [P.IsRootSystem]
+
+/-- The polarization map from weight space to coweight space as an equivalence. -/
+def PolarizationEquiv : M ≃ₗ[R] N :=
+  have : IsReflexive R M := Module.IsReflexive.of_isPerfPair P.toLinearMap
+  (P.toInvariantForm.form.toDual P.rootForm_nondegenerate).trans P.flip.toPerfPair.symm
+
+@[simp]
+lemma polarizationEquiv_toLinearMap :
+    P.PolarizationEquiv.toLinearMap = P.Polarization := by
+  simp only [PolarizationEquiv, LinearMap.BilinForm.toDual, RootPairing.toInvariantForm_form,
+    ← P.flip_comp_polarization_eq_rootForm, RootPairing.flip_toLinearMap]
+  ext m
+  let e := P.flip.toPerfPair
+  change e.symm (e _) = _
+  simp
+
+lemma polarizationEquiv_apply (m : M) :
```

Comment:
Should this be simp, or did you leave the attr out for a reason? (If so, maybe add a short comment.)
```suggestion
@[simp]
lemma polarizationEquiv_apply (m : M) :
```